### PR TITLE
fix(controls): color correction on contols

### PR DESCRIPTION
**Pull request by hormaza**
Focus indicators have contrast issues:
image

This commit will increase the background alpha on the toolbar at 90% and increase the brightness of the focus indicators at #D3D3D3 (box-gray-20)

### CODE REVIEW

File: src/lib/viewers/controls/_styles.scss [hunk 1]
```diff
@@ -2,8 +2,8 @@
 @import '~box-ui-elements/es/styles/deprecated';
 
 $bp-controls-background: fade-out($black, .2);
-$bp-controls-opacity: .8;
-$bp-controls-outline: inset 0 0 0 1px fade-out($white, .5), 0 1px 2px fade-out($black, .9);
+$bp-controls-opacity: .9;
+$bp-controls-outline: inset 0 0 0 1px fade-out($bdl-gray-20, 0), 0 1px 2px fade-out($black, .9);
```
Comment (jstoffan) [inline src/lib/viewers/controls/_styles.scss:6]: We should be able to remove the `fade-out` directive here, since we're passing a zero value to it.

**Comment by jstoffan** [comment]
@hormaza, can you please include both a "before" and "after" screenshot for all visual changes to preview? Thanks!

File: src/lib/viewers/controls/_styles.scss (hunk 1 above)
Comment (hormaza) [reply src/lib/viewers/controls/_styles.scss]: Thanks Jared. Shouldn't be zero. Updated!

**Comment by hormaza** [comment]
Before:
image
After:
image

**Review by ConradJChan — APPROVED** [review]

**Pull request merged**